### PR TITLE
Update abis, bump fuels version

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/sdk/setPrimaryDomain.ts [hunk 1]
```diff
@@ -13,10 +13,13 @@ const setPrimaryDomain = async (wallet: Account, domain: string, txParams: TxPar
     const resolverContractInstance = getResolverContract(wallet);
     const invocationScope = resolverContractInstance
       .functions
-      .set_primary({bits: assetId})
+      .set({bits: assetId})
```
Comment (fuelnamedev) [inline src/sdk/setPrimaryDomain.ts]: This call doesn't set the primary domain. You should call `set_primary` function from the registry contract

**Review by fuelnamedev — CHANGES_REQUESTED** [review]

File: src/sdk/setPrimaryDomain.ts (hunk 1 above)
Comment (ghost) [reply src/sdk/setPrimaryDomain.ts]: Fixed

File: src/sdk/setPrimaryDomain.ts [hunk 2]
```diff
@@ -10,13 +10,16 @@ const setPrimaryDomain = async (wallet: Account, domain: string, txParams: TxPar
     }
     const assetId = assetIdResult.value!;
 
-    const resolverContractInstance = getResolverContract(wallet);
-    const invocationScope = resolverContractInstance
+    const registryContractInstance = await getRegistryReadOnlyContract();
```
Comment (fuelnamedev) [inline src/sdk/setPrimaryDomain.ts]: This shouldn't be readonly, cause we're writing onchain

**Review by fuelnamedev — CHANGES_REQUESTED** [review]

**Review by fuelnamedev — APPROVED** [review]

File: src/sdk/setPrimaryDomain.ts (hunk 2 above)
Comment (ghost) [reply src/sdk/setPrimaryDomain.ts]: Fixed

**Pull request merged**